src/css/forms.scss

```diff
@@ -369,7 +369,7 @@ p.sentence-case:first-letter {
     margin-right: 10px;
 }
 
-.form__radio-group input {
+.form__radio-group input[type="radio"] {
```

Comment:
Confirmed; this class is being used for radio group only. This was causing the styling of input in the radio group to break